fleet/lib/rook-ceph-cluster/values.yaml

```diff
@@ -117,17 +117,17 @@ cephClusterSpec:
         memory: 512Mi
     mon:
       limits:
-        cpu: "2"
+        cpu: "1"
```

Comment:
I would rather leave the mons request/limit as-is.  If the mon becomes CPU limited all I/O operations slow down.